scripts/world_bank/worldbank.py

```diff
@@ -139,15 +160,16 @@ def row_to_constraints(row):
             next_constraint += 1
         return constraints_text
 
+    # TODO(tjann): try to turn this into a format_map.
     # yapf: disable
     # Input all required statistical variable fields.
-    new_stat_var = (TEMPLATE_STAT_VAR
-        .replace("{INDICATOR}", row['IndicatorCode'].replace(".", "_"))
-        .replace("{NAME}", row['IndicatorName'])
-        .replace("{DESCRIPTION}", row['SourceNote'])
-        .replace("{measuredProperty}", row['measuredProp'])
-        .replace("{CONSTRAINTS}", row_to_constraints(row))
-    )
+    new_stat_var = (TEMPLATE_STAT_VAR.replace(
```

Comment:
might think how to do this, its not quite readable now...

Reply:
Yeah I agree. See my comment above